tokens/custom/large-vars.css

```diff
@@ -131,4 +131,9 @@
 	/* XXL and XXS icon sizes are not within the design spec and are planned to be deprecated in Spectrum 2. */
 	--spectrum-workflow-icon-size-xxl: 40px;
 	--spectrum-workflow-icon-size-xxs: 15px;
+
+	--spectrum-in-field-button-side-edge-to-fill-small: 3px;
+	--spectrum-in-field-button-side-edge-to-fill-medium: 4px;
+	--spectrum-in-field-button-side-edge-to-fill-large: 4px;
+	--spectrum-in-field-button-side-edge-to-fill-extra-large: 4px;
```

Comment:
This set matches the mobile values list in Figma, so I think just the medium-vars need to be updated. 👍